site/docs/pages/index.mdx

```diff
@@ -119,20 +122,26 @@ bun add @coinbase/onchainkit
     <div className="flex grow flex-col md:flex-row">
       <div>
 ```tsx
-      <Wallet> // [!code focus]
-        <ConnectWallet> // [!code focus]
-          <Avatar className="h-6 w-6" />
-          <Name />
-        </ConnectWallet> // [!code focus]
-        <WalletDropdown> // [!code focus]
-          <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick>
-          </Identity>
-          <WalletDropdownLink> // [!code focus]
-            Go to Wallet Dashboard // [!code focus]
-          </WalletDropdownLink> // [!code focus]
-          <WalletDropdownDisconnect /> // [!code focus]
-        </WalletDropdown> // [!code focus]
-      </Wallet> // [!code focus]
+<Wallet> // [!code focus]
```

Comment:
@cpcramer spaces matters in code blocks. Make sure those are added in respect on how best it looks on the site.